app/RequirementTests/CourseControllerTests.hs

```diff
@@ -0,0 +1,75 @@
+module RequirementTests.CourseControllerTests
+( courseContTestSuite ) where
+
+import Config (runDb)
+import Data.Text as T
+import qualified Data.ByteString.Lazy.Char8 as BL
+import qualified Data.Map as Map
+import Test.HUnit
+import Happstack.Server (HttpVersion(..), Request(..), simpleHTTP'', rsBody, Response, ServerPart, Method(GET))
+import Control.Concurrent.MVar (newMVar, newEmptyMVar)
+import Database.Database (clearDatabase)
+import Database.Persist.Sqlite (SqlPersistM, insert_)
+import Database.Tables
+import Controllers.Course (index)
+
+-- | A minimal mock request for running a ServerPart
```

Comment:
Move this function and the one below (`runServerPart`), and the `clearDatabase` function, into a new test helper module. This is how you can specify functions that are intended to be used across multiple test suites in different files.